Apps/WebClient/src/ClientApp/src/components/modal/rating.vue

```diff
@@ -61,8 +63,16 @@ export default class RatingComponent extends Vue {
     private ratingValue: number = 0;
     private isVisible: boolean = false;
 
+    @Getter("webClient", { namespace: "config" })
+    config!: WebClientConfiguration;
+
     public showModal() {
         this.isVisible = true;
+        setTimeout(() => {
+            if (this.isVisible) {
+                this.handleRating(0, true);
```

Comment:
just to check with mandad and nino that timing out would be the same as skipping